process/block/shardblock.go

```diff
@@ -346,7 +347,7 @@ func (sp *shardProcessor) ProcessBlock(
 		return err
 	}
 
-	return nil
+	return sp.blockProcessingCutoffHandler.HandleProcessErrorCutoff(header)
```

Comment:
Let's change this to:
```GO
err = sp.blockProcessingCutoffHandler.HandleProcessErrorCutoff(header)
if err != nil {
    return err
}

return nil
```

`err` is evaluated in the defer anonymous function

Reply:
done